source/userguide/dvc.rst

```diff
@@ -29,6 +36,7 @@ Available Options in MindMeld
 | --help          | Show the available options and their descriptions.                     |
 +-----------------+------------------------------------------------------------------------+
 
+[1] The DAGsHub central repository requires you `sign up to DAGsHub <https://dagshub.com/user/sign_up>`_ (it's completely free), and create a project – you can `connect an existing GitHub project <https://dagshub.com/repo/connect>`_ or `create a project from scratch <https://dagshub.com/repo/create>`_.
```

Comment:
1. (it's completely free) -> (it's free for personal projects)
2. Could you please also mention https://dagshub.com/repo/migrate to migrate existing projects?
3. Could you also mention `setup_dagshub ` only works once  dvc project has been initialized in the repo?

Reply:
Done!